components/menu/style/index.less

```diff
@@ -486,7 +537,7 @@
         line-height: @menu-item-height;
         + span {
           display: inline-block;
-          max-width: 0;
+          // max-width: 0;
```

Comment:
Hey @zombieJ, this commented line is causing alignment issues in our application. I am wondering why this line is uncommented instead removed. Is this a mistake and do you planning to uncomment it?

Reply:
Mistake...Let me remove this. Thx